hs_core/hydroshare/resource.py

```diff
@@ -425,6 +433,16 @@ def create_resource(
         if not metadata:
             metadata = []
 
+        fed_zone_home_path = ''
+        if fed_res_path:
```

Comment:
@pkdash If I understand your question correctly, that is not the use case being addressed here, but the main use case to be addressed here is creating a new version of a resource from an existing resource, in which case, the resource_federation_path of the existing resource needs to be passed in to create_resource() so that the new resource will be created in the same federated zone as the original resource. There are other use cases as well that need to create resource in a specified federated zone which can be achieved by passing in an fed_res_path parameter so that the resource can be stored in the specified federated zone. Hope this answers your question here.